### PR TITLE
SUBS 1391

### PR DESCRIPTION
**Pull request by davidrichardson**
This updates the ena core service library, so that it converts sequence variation analysis documents into ENA Analysis objects, submits them via the ENA API and recovers the accessions from the receipt

### CODE REVIEW

File: src/main/resources/uk/ac/ebi/subs/data/submittable/seq_var_analysis_mapping.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <xml-root-element name="ANALYSIS"/>
+            <xml-type prop-order="accession alias teamName title description studyRefs sampleRefs
+                assemblyName experimentType programs platforms imputation files enaAttributeList"/>
+            <java-attributes>
+                <xml-attribute java-attribute="accession"/>
+                <xml-attribute java-attribute="alias"/>
+                <xml-attribute java-attribute="teamName" name="center_name"/>
+                <xml-element java-attribute="accession" name="IDENTIFIERS/PRIMARY_ID"/>
+                <xml-element java-attribute="alias" xml-path="IDENTIFIERS/SUBMITTER_ID/text()"/>
+                <xml-attribute java-attribute="teamName" xml-path="IDENTIFIERS/SUBMITTER_ID/@namespace"/>
```
Comment (ke4) [inline src/main/resources/uk/ac/ebi/subs/data/submittable/seq_var_analysis_mapping.xml]: This is the 2nd teamName attribute

**Review by ke4 — CHANGES_REQUESTED** [review]
A tiny comment, but otherwise looks cool 👍

File: src/main/resources/uk/ac/ebi/subs/data/submittable/seq_var_analysis_mapping.xml (hunk 1 above)
Comment (davidrichardson) [reply src/main/resources/uk/ac/ebi/subs/data/submittable/seq_var_analysis_mapping.xml]: good catch, I copied that from the existing mappings. I've fixed it in the mappings for all submittables

**Review by ke4 — APPROVED** [review]
👍

**Pull request merged**